proto/schema/v1/schema.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package schema.v1;
+
+service SchemaService {
+  rpc Register(RegisterRequest) returns (RegisterResponse);
+  rpc Fetch(FetchRequest) returns (FetchResponse);
```

Comment:
I propose sticking to the same terminology we use for example in the [Conduit API](https://github.com/ConduitIO/conduit/blob/main/proto/api/v1/api.proto), i.e. `Create`, `Get`, `Update`, `Delete`.

Reply:
Sounds good!